recipe/meta.yaml

```diff
@@ -22,25 +22,31 @@ requirements:
     - wheel
   run:
     - python
-    - async_generator >=1.9
-    - attrs >=19.2.0
+    - attrs >=20.1.0
     - cffi >=1.14  # [win]
-    - contextvars >=2.1  # [py<37]
     - idna
     - outcome
-    - sniffio
+    - sniffio >=1.3.0
     - sortedcontainers
-    - exceptiongroup >=1.0.0rc9  # [py<311]
+    - exceptiongroup # [py<311]
 
 test:
+  source_files:
+    - pyproject.toml
   imports:
     - trio
     - trio.testing
-    - trio.tests
   requires:
     - pip
+    - pytest
+    - async_generator >= 1.9
+    - astor
+    - isort
+    - pyopenssl >= 22.0.0   # for the ssl + DTLS tests
+    - trustme               # for the ssl + DTLS tests
   commands:
     - pip check
+    - python -m pytest --pyargs trio -ra --durations=10 -m "not redistributors_should_skip" --skip-optional-imports
```

Comment:
comment about why -ra and --durations-10 are there could be useful

Reply:
-ra gives a short summary of all except pass tests.
--durations gives the slowest 10 durations.
Upstream is using those flags, we could do without. But anycase, these are standard pytest arguments.

Reply:
I was wondering why we need e.g. the slowest 10 durations. If it's just taken from upstream, fine